### DISCUSSION TITLE
How to use u8g2 with SSD1306 OLED videos!

### DISCUSSION THREAD

**Discussion by stonez56** [olikraus/u8g2#2061] [Show and tell]
Hi All, 
I've made how to use u8g2 library with ESP32 + SSD1306 0.9" OLED in a YouTube channel.
[Chinese version](https://youtu.be/Ksdwd87JpPs)
[English version](https://youtu.be/-bntyE8UYTY)
 
Hope these are helpful to u8g2 beginners!
Any comment is welcome!

![image](https://user-images.githubusercontent.com/5953052/209415144-1dd8bd1c-6157-4322-8a58-69031d25016f.png)

Stonez56

**Comment by olikraus** [comment]
Thanks for sharing the video. Very helpful.

**Reply by stonez56** [reply]
Thanks! :)
